### PR TITLE
Fix adding hooks import and removing React Import

### PR DESCRIPTION
**Pull request by IlyaKhD**

### CODE REVIEW

File: packages/inferno-from-react-generator/src/__tests__/react-test-cases/import-without-react.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import React, { useEffect, useMemo } from 'react';
+
+//* Component={"name":"DxComponent"}
+function DxComponent() {
+  const x = useEffect.toString() + useMemo.toString();
```
Comment (williamvinogradov) [inline packages/inferno-from-react-generator/src/__tests__/react-test-cases/import-without-react.tsx:5]: Minor comment

Let's name it ```className```, but in test, it's not important

File: packages/inferno-from-react-generator/src/__tests__/test-cases/import-without-react.jsx [hunk 2]
```diff
@@ -0,0 +1,7 @@
+import React, { useEffect, useMemo } from 'react';
+//* Component={"name":"DxComponent"}
+function DxComponent() {
+    const x = useEffect.toString() + useMemo.toString();
```
Comment (williamvinogradov) [inline packages/inferno-from-react-generator/src/__tests__/test-cases/import-without-react.jsx:4]: Minor comment

Let's name it ```className```, but in test, it's not important

File: packages/inferno-from-react-generator/src/expressions/imports-aggregation.ts [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+  constructor(...declarations: AgregateableImportDeclaration[]) {
+    this.declarations = declarations;
+  }
+
+  toString(): string {
+    return this.declarations.map((d) => d.toString()).join('\n');
+  }
+
+  replaceSpecifier() {
```
Comment (williamvinogradov) [inline packages/inferno-from-react-generator/src/expressions/imports-aggregation.ts:33]: Question.

Maybe I missed something.
But, why do we need these not implemented methods in this class?

**Review by williamvinogradov — APPROVED** [review]

File: packages/inferno-from-react-generator/src/expressions/imports-aggregation.ts (hunk 3 above)
Comment (IlyaKhD) [reply packages/inferno-from-react-generator/src/expressions/imports-aggregation.ts]: I've made this in order to fit ImportDeclaration type and avoid implementing these methods. implementation is not required and even more - not obvious. E.g. what should the `add` method regarding multiple import clauses?

Whatever, @MikeVitik  asked me to rewrite this part, take a look at the new one

File: packages/inferno-from-react-generator/src/__tests__/test-cases/import-without-react.jsx (hunk 2 above)
Comment (IlyaKhD) [reply packages/inferno-from-react-generator/src/__tests__/test-cases/import-without-react.jsx]: done

File: packages/inferno-from-react-generator/src/__tests__/react-test-cases/import-without-react.tsx (hunk 1 above)
Comment (IlyaKhD) [reply packages/inferno-from-react-generator/src/__tests__/react-test-cases/import-without-react.tsx]: done

**Review by MikeVitik — APPROVED** [review]

**Pull request merged**